scipy/special/basic.py

```diff
@@ -754,20 +754,97 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse error function.
+    This function compute the inverse error function.
+    Inverse error function is satisfying :math:`erf(erf^{-1}(x))=x`.
+
+    Parameters
+    ----------
+    y : ndarray
+        Argument at which to evaluate for (-1, 1)
```

Comment:
What does `evaluate for (-1, 1)` intended to indicate here?  Is the domain the open interval `(-1, -1)`? Or perhaps the closed interval `[-1, 1]`?

Reply:
`evaluate for (-1, 1)` means the domain the open interval from -1 to 1.
Should I depict more clearly? Could you recommend the expressions?

Reply:
Saying "evaluate for" itself is not very clear, that's not a construct in English that I recognize.

You could e.g. write "Argument at which to evaluate. Domain: (-1, 1)" or something similar.